### ISSUE TITLE
stylelint indentation rule broke in v6.6.0

### ISSUE THREAD

**Issue by ipinak** [stylelint/stylelint#1494]
After upgrading to from `6.5.1` to `6.6.0` I got errors on indentation related to sass maps.

> The issue is related to `"indentInsideParens": "once"` rule.

According to the above rule the following code should be valid.

```
$colors: (
	primary: #387ef5,
	secondary: #32db64,
	danger: #f53d3d,
	light: #f4f4f4,
	dark: #222,
	favorite: #69bb7b
);
```

> The following configuration is used, along with a lot of other rules, but this is the one that causes the issue.

```json
"indentation": [
	"tab",
	{
		"indentClosingBrace": false,
		"indentInsideParens": "once"
	}
],
```

> Which version of stylelint are you using?

`6.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

I've used a postcss plugin where one of the processors is stylelint, like the example below.

```javascript
var processors = [
	stylelint(),
	postcssReporter({
		clearMessages: true,
		throwError: true
	})
];

return gulp.src(SASS_SRC_FILES)
	.pipe(postcss(processors, {
		syntax: postcssScss
	}));
```

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to SCSS maps.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
app/theme/app.variables.scss
26:2    ‼  Expected indentation of 0 tabs (indentation) [stylelint]
27:2    ‼  Expected indentation of 0 tabs (indentation) [stylelint]
28:2    ‼  Expected indentation of 0 tabs (indentation) [stylelint]
29:2    ‼  Expected indentation of 0 tabs (indentation) [stylelint]
30:2    ‼  Expected indentation of 0 tabs (indentation) [stylelint]
31:2    ‼  Expected indentation of 0 tabs (indentation) [stylelint]

[10:34:26] 'lint:sass' errored after 93 ms
[10:34:26] Error in plugin 'gulp-postcss'
Message:

** postcss-reporter: warnings or errors were found **
Details:
    fileName: C:\app\theme\app.variables.scss
Stack:
Error:
** postcss-reporter: warnings or errors were found **
    at C:\node_modules\postcss-reporter\lib\reporter.js:47:13
    at LazyResult.run (C:\node_modules\postcss\lib\lazy-result.js:206:20)
    at C:\node_modules\postcss\lib\lazy-result.js:120:37
    at LazyResult.asyncTick (C:\node_modules\postcss\lib\lazy-result.js:134:15)
    at C:\node_modules\postcss\lib\lazy-result.js:125:31
    at process._tickCallback (node.js:406:9)
```

I have verified that the rule works just fine in `stylelint#6.5.1`.

**Comment by jeddy3** [comment]
@ipinak Thanks for the detailed write up. This was addressed in https://github.com/stylelint/stylelint/pull/1493 and will be in the next release.

@davidtheclark Shall I get out `6.6.1` out today with the three fixes in. I can also bump `postcss-less`.

**Comment by ipinak** [comment]
Thanks a lot.

**Comment by davidtheclark** [comment]
@jeddy3 Yeah, no reason I know of to delay on a release.

**Comment by jeddy3** [comment]
@ipinak Released as `6.7.0`.

**Comment by ipinak** [comment]
Just checked and it seems that it still does not work. I get the same error.

**Comment by jeddy3** [comment]
@ipinak Can you double check that you're using the latest version please?:

```console
stylelint --version
./node-modules/.bin/stylelint --version
```

I've [just added passing tests](https://github.com/stylelint/stylelint/pull/1503) for your configuration and use-case. If you're definitely using the latest version, then can you please send us a PR with failing tests as we're going to struggle to resolve this otherwise. Thanks!

**Comment by ipinak** [comment]
Double checked it's `6.7.0`

$ ./node_modules/.bin/stylelint --version
6.7.0

**Comment by jeddy3** [comment]
Ok, thanks for double checking. This is a head-scratching one :)

The next step is to submit a PR with failing tests. You can use [my PR](https://github.com/stylelint/stylelint/pull/1503) as a template. All I can of is that there might be something other in your `app.variables.scss`, that I didn't account for in my PR, that is causing the false positives. Here are [the instructions](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#running-tests) to run the tests locally.

**Comment by evilebottnawi** [comment]
@jeddy3 work fine with new version for my issue.

**Comment by ipinak** [comment]
@jeddy3 the `app.variables.scss` contains only that map. I will try to look into it a bit more and if I find something I will create a PR.
